<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buttonBox</class>
 <widget class="QWidget" name="buttonBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>160</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Gilroy</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Добавление врага</string>
  </property>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>122</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gilroy</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>122</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gilroy</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fraction_box">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Выберите группировку</string>
   </property>
  </widget>
  <widget class="QLabel" name="fraction_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gilroy</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Группировка:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="difficult_box">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Выберите сложность</string>
   </property>
  </widget>
  <widget class="QLabel" name="difficult_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gilroy</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сложность:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="wealth_box">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Выберите богатство</string>
   </property>
  </widget>
  <widget class="QLabel" name="wealth_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gilroy</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Богатство:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
